app/globals.css

```diff
@@ -115,5 +115,6 @@
 
 .mapboxgl-popup-content {
   padding: 0 !important;
+  background-color: transparent !important;
```

Comment:
This is to override the default mapbox value

Reply:
Let's ask for team feedback on Thursday, I feel this makes the text difficult to read but it could just be me.